client/src/components/manageProjects/editMeetingTimes.js

```diff
@@ -39,7 +39,8 @@ const EditMeetingTimes = ({
         };
       }
 
-      if (values.description) {
+      //if there is a description or a blank description
+      if (values.description || !values.description) {
```

Comment:
Help us understand why the if statement is necessary here? Wouldn't this conditional just always evaluate to `true` ?

Reply:
Yes, it needs to always be true so the description is carried over with the update. When there is a blank description, the original conditional, `if (values.description)`, will be falsy so the blank description doesn't get updated. I guess there is a better way to rewrite this file in which we do a catch all to update all the values like:
`theUpdatedEvent = {
          ...theUpdatedEvent,
          **value name: updated value**
};`